app/assets/stylesheets/events.scss

```diff
@@ -27,6 +23,34 @@ hr {
   margin-bottom: 20px;
 }
 
+.event-card {
+  padding: 25px;
+  margin-bottom: 25px;
+}
+
+.card-title {
+  font-size: 40px;
+  color: white;
+}
+
+.card-text {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.fab .fa-facebook {
+  width: 24px !important;
+  height: 24px !important;
+}
+
```

Comment:
I think you messed up the merge conflict resolution. these were removed and put in application.scss because more pages were using them.